jupyter_server_proxy/standalone/__init__.py

```diff
@@ -0,0 +1,214 @@
+    )
+    parser.add_argument(
+        "-s",
+        "--server-port",
+        default=0,
+        type=int,
+        dest="server_port",
+        help="Port for the WebApp should end up running on (0 for random open port).",
+    )
+    parser.add_argument(
+        "--socket-path",
+        type=str,
+        default=None,
+        help="Path to the Unix Socket to use for proxying. Takes precedence over '-s/--server_port' and '--socket-auto'.",
+    )
+    parser.add_argument(
+        "--socket-auto",
+        action="store_true",
+        help="Use Unix Socket for proxying, but let Jupyter Server Proxy automatically create one.",
+    )
+    parser.add_argument(
+        "--env",
+        "--environment",
+        type=lambda v: tuple(v.split(":")[:2]),
+        default=[],
+        action="append",
+        dest="environment",
+        help="Add an environment variable to the server process. Must be of the form <Name>:<Value>, e.g. --env=MY_VAR:42",
+    )
+    parser.add_argument(
+        "--mappath",
+        type=lambda v: tuple(v.split(":")[:2]),
+        default=[],
+        action="append",
+        help="Add an path mapping to the proxy. Any requests received under <Source> will be redirected to <Target>. "
+        "Must be of the form <Source>:<Target>, e.g. --mappath=/:/index.html",
+    )
+    parser.add_argument(
+        "-d",
+        "--debug",
+        action="store_true",
+        default=False,
+        dest="debug",
+        help="Display debug level logs.",
+    )
+    # ToDo: Split Server and Application Logger
+    # parser.add_argument(
+    #     "--logs",
+    #     action="store_true",
+    #     default=True,
+    #     help="Display logs generated by the subprocess.",
+    # )
+    parser.add_argument(
+        "--skip-authentication",
+        action="store_true",
+        help="Do not enforce authentication with the JupyterHub Server.",
+    )
+    parser.add_argument(
+        "--timeout",
+        default=60,
+        type=int,
+        help="Timeout to wait until the subprocess has started and can be addressed.",
+    )
+    parser.add_argument(
+        "--activity-interval",
+        default=300,
+        type=int,
+        help="Frequency to notify Hub that the WebApp is still running (In seconds, 0 for never).",
+    )
+    # ToDo: Progressive Proxy
+    # parser.add_argument(
+    #     "--progressive",
+    #     action="store_true",
+    #     default=False,
+    #     help="Progressively flush responses as they arrive (good for Voila).",
+    # )
+    parser.add_argument(
+        "--websocket-max-message-size",
+        default=0,
+        type=int,
+        help="Max size of websocket data (leave at 0 for library defaults).",
+    )
+    parser.add_argument(
+        "command", nargs="+", help="The command executed for starting the WebApp"
+    )
+
+    args = parser.parse_args()
```

Comment:
There's a mismatch between the options supported by Jupyter Server Proxy
https://github.com/jupyterhub/jupyter-server-proxy/blob/v4.4.0/jupyter_server_proxy/config.py#L34-L47
and those available here. Obviously some options aren't relevant, but for example `raw_socket_proxy` is.